### ISSUE TITLE
Add Confirmation Prompt to Z-Wave Advanced Settings

### ISSUE THREAD

**Issue by PascalTurbo** [openhab/org.openhab.ui.habmin#237]
For the Advanced Settings in Z-Wave Configuration it's possible to select destructive options like "Hard Reset Controller" without any security prompt. So with a single click a user is able to destroy it's whole Z-Wave network by accident. 

I would prefer a dialog which tells the user what will happen and forces him to confirm its selection.

**Comment by cdjackson** [comment]
I would agree but I don't know w how we can define this. There is no such option in ESH so I don't think this is possible - sorry.

**Comment by PascalTurbo** [comment]
Maybe we could create a feature request to ESH?

**Comment by cdjackson** [comment]
I thought about this after your issue request and decided it was a good move so I already made a PR…

https://github.com/eclipse/smarthome/pull/2258 <https://github.com/eclipse/smarthome/pull/2258>

I need to tidy it up some more over the next week or so...
